charts/operator-wandb/smooth-template.yaml

```diff
@@ -0,0 +1,2834 @@
+---
```

Comment:
What purpose does this file serve?  It looks to be autogenerated, so I'd be wary of including it here.

Reply:
Yah, that was my bad.

Reply:
Rotated that license.